backend/app/crud/prompt_version.py

```
@@ -0,0 +1,273 @@
+    """
+    prompt_versions = get_prompt_versions(
+        session=session,
+        prompt_id=prompt_id,
+        project_id=project_id,
+        skip=skip,
+        limit=limit,
+    )
+
+    # make sure to prompt_id is valid and not deleted
+    stmt = select(func.count()).where(
+        PromptVersion.prompt_id == prompt_id, PromptVersion.is_deleted == False
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not PromptVersion.is_deleted:` for false checks

Replace with `not PromptVersion.is_deleted`

(E712)